src/common/definitions.js

```diff
@@ -692,6 +851,15 @@ const uuid = {
   format: 'uuid',
 };
 
+const hcaEmail = {
+  type: 'string',
+  maxLength: 256,
+  format: 'email',
+  // regex from client/validations.js' isValidEmail, with some extra escaping
+  pattern:
```

Comment:
you don't need to put "format: email" and add a regex, putting "format: email" alone will validate that the string is an email

Reply:
Okay, I'll change this back then. Thanks!

Reply:
So for some reason, `format: 'email'` was allowing `a@a` through, so I'll just use the pattern we use in the EZ
![](https://github.com/department-of-veterans-affairs/vets-json-schema/assets/34111449/d4a84896-7e53-41e5-965f-a37aca6ccb7d)